lib/vmdb/appliance.rb

```diff
@@ -107,70 +107,73 @@ def self.log_diagnostics
       end
     end
 
-    private
-
-    def self.get_build
-      build_file = File.join(File.expand_path(Rails.root), "BUILD")
+    # start of private class methods
+    class << self
```

Comment:
Not a big fan of this syntax.  I think it makes it harder to grep for class methods.  /cc @Fryguy